src/component/forcefield/MechanicalMatrixMapperMOR.h

```diff
@@ -16,6 +16,7 @@
 ******************************************************************************/
 #ifndef MECHANICALMATRIXMAPPERMOR_H
 #define MECHANICALMATRIXMAPPERMOR_H
+#include <ModelOrderReduction/initModelOrderReduction.h>
```

Comment:
isn't it useless here ?

Reply:
Possible, I added it everywhere just in case :small_airplane: